### PR TITLE
Clean up README, update links from vaikas/sigstore-scaffolding to sigstore/scaffolding.

### PR DESCRIPTION
**Pull request by vaikas**
update links / mentions of vaikas/sigstore-scaffolding to
sigstore/scaffolding

Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

#### Ticket Link

Fixes

#### Release Note

```release-note

```

### CODE REVIEW

**Review by n3wscott — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/vaikas/sigstore-scaffolding
+module github.com/sigstore/scaffolding
 
 go 1.16
```
Comment (cpanato) [inline go.mod:3]: why not move straight to 1.17?

**Review by cpanato — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (vaikas) [reply go.mod:3]: I don't think I use anything that would require go 1.17 as the min version. But I can change if you think it makes sense?

Comment (cpanato) [reply go.mod]: this project is not meant to be consumable as a lib right?

Comment (vaikas) [reply go.mod]: Yeah. So I changed it :)

**Pull request merged**